LumpNav/LumpNav.py

```diff
@@ -115,7 +115,7 @@ def addValuesToDefaultConfiguration(self):
                    'TipToSurfaceDistanceTrajectory' : 'True',
                    'NeedleModelToNeedleTip' : '0.0 1.0 0.0 0.0 0.0 0.0 1.0 0.0 1.0 0.0 0.0 0.0 0 0 0 1',
                    'NeedleBaseToNeedle' : '1 0 0 20.93 0 1 0 14.00 0 0 1 -4.27 0 0 0 1',
-                   'CauteryModelToCauteryTip' : '0 0 1 0 0 -1 0 0 1 0 0 0 0 0 0 1',
+                   'CauteryModelToCauteryTip' : '0.03 0.03 1.01 0.00 -0.23 0.98 -0.02 0.00 -0.97 -0.23 0.03 0.00 0 0 0 1', 
```

Comment:
Do you know why you have value >1.0 in the rotation part of the matrix?
(it's not much, only 1.01, so probably the magnification is not visible, I'm just wondering what could have happened)